ui/pages/build-status.js

```diff
@@ -88,6 +88,22 @@ const RepoStatusItem = ({ repo, organization = 'Mender', branch = 'master', cove
   </Stack>
 );
 
+const buildStatusColor = (status) => {
+    switch(status) {
+    case 'FAILED':
+        return 'error';
+        break;
+    case 'RUNNING':
+        return 'inprogress';
```

Comment:
`inprogress` doesn't seem to be part of the [mui palette](https://mui.com/material-ui/customization/palette/)